### ISSUE TITLE
Missing label for the object selector dropdown in properties panel

### ISSUE THREAD

**Issue by tsv2013** [surveyjs/survey-creator#870]
See the https://surveyjs.answerdesk.io/ticket/details/t4494/accessibility-issue-for-pagination-dropdown-ensure-every-form-element-has-a-label thread for more details.
